lib/resque/failure/sentry.rb

```diff
@@ -16,26 +17,26 @@ module Failure
     #   Resque::Failure.backend = Resque::Failure::Multiple
     #
     class Sentry < Base
-
-      def self.logger
-        @logger
+      class << self
+        attr_reader :logger
       end
 
-      def self.logger=(value)
-        @logger = value
+      class << self
+        attr_writer :logger
       end
 
       def save
         options = {}
         options[:logger] = self.class.logger if self.class.logger
         options[:extra] = {
-          "Object" => payload['class'],
-          "Arguments" => payload['args']
+          'Object' => payload['class'],
+          'Arguments' => payload['args']
         }
-        Raven.capture_exception(exception, options)
+        options[:hint] = { background: false }
```

Comment:
@samuelgiles The culprit! I'm not exactly sure _why_ this doesn't work (it defaults to true) but explicitly setting it to false restores the pre-update behaviour